app/models/bibliography_resource.rb

```diff
@@ -2,4 +2,6 @@
 # BibliographyResource model class
 class BibliographyResource < Spotlight::Resource
   self.document_builder_class = BibliographyBuilder
+
+  store :data, accessors: [:bibtex_file]
```

Comment:
Wondering if we would get some benefit using `CarrierWave` instead of storing the raw data here.

Reply:
I've add an `accept: 'application/x-bibtex'` to the form, but I'd like to stay away from carrierwave at this time. It feels like maybe a level of indirection we don't need.